src/Engine/UnitActionType.java

```diff
@@ -44,7 +44,7 @@ public GameActionSet getPossibleActions(GameMap map, Path movePath, Unit actor)
     @Override
     public GameActionSet getPossibleActions(GameMap map, Path movePath, Unit actor, boolean ignoreResident)
     {
-      XYCoord moveLocation = new XYCoord(movePath.getEnd().x, movePath.getEnd().y);
+      XYCoord moveLocation = movePath.getEndCoord();
```

Comment:
I feel like maybe this should be moved into a parent function or something, so it isn't copy-pasted everywhere?
*shrug*

Reply:
_shrug_